packages/webviz-core/src/panels/ThreeDimensionalViz/LaserScanVert.js

```diff
@@ -38,7 +39,7 @@ export default `
     } else if (isHitmap) {
       vColor = hitmapColor;
     } else {
-      vColor = vec4(0.5, 0.5, 1, 1);
+      vColor = overrideColor;
```

Comment:
Do we need to handle case for when `overrideColor` is not defined?

Also, should we just use `color` for topic settings going forward, since we are treating both the same way, and the 2 names have caused issues in the past?

Reply:
I still like `overrideColor ` over `color` since it's more explicit.  Could definitely consolidate into one later though.